experiments/colgranule/vector_graph_deep1b_bench_test.go

```
@@ -0,0 +1,772 @@
+	}
+	cacheDir, err := os.UserCacheDir()
+	if err != nil {
+		tb.Fatalf("UserCacheDir: %v", err)
+	}
+	return filepath.Join(cacheDir, "gomap", "deep1b")
+}
+
+func columnVectorGraphDeep1BDownloadFbin(tb testing.TB, path string, url string, prefixRows int, dims int) error {
+	tb.Helper()
+	var expectedBytes int64
+	req, err := http.NewRequest(http.MethodGet, url, nil)
```

golangci-lint (2.12.2)

[error] net/http.NewRequest must not be called. use net/http.NewRequestWithContext

(noctx)